data/coalition/coalition missions.txt

```diff
@@ -1551,49 +1553,62 @@ mission "Saryd University Lecture"
 				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they have a lot of autonomy."`
-			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+			`	As you answer the question, your translator begins to cut out. Repeating your answer does no good, as it contiunes to fizzle out as you talk. You look over to the Heliarch, who eyes the curious Kimek with suspicion. "Apologize for the malfunction, I must," says the Heliarch. "The next question, answer it please."`
 				goto questionthree
 			label war
 			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
 			choice
 				`	"Yes, all of human space is united under the Republic."`
+					goto united
 				`	"Yes, but there are internal subdivisions: the Free Worlds in the south, the Republic in the north, and the Syndicate in the east."`
 				`	"No. Technically all of human space is under the Republic, but the Free Worlds and Syndicate have significant autonomy."`
-			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+			`	As you answer the question, your translator begins to cut out. Repeating your answer does no good, as it continues to fizzle out as you talk. You look over to the Heliarch, who eyes the curious Kimek with suspicion. "Apologize for the malfunction, I must," says the Heliarch. "The next question, answer it please."`
+				goto questionthree
+			label united
+			`	Upon finishing your sentence, several in the crowd begin to write down notes on datapads. It appears that information on human government is new to them.`
 			label questionthree
-			`	The Arach picks another Kimek. "Still a problem, is piracy in human space?"`
+			`	The Arach picks another Kimek. "Piracy in human space, how big of a problem is it?"`
 			choice
-				`	"Yes, there are pirate planets on the fringes of human space."`
+				`	"There are entire pirate planets on the fringes of human space where warlords and criminals rule."`
 				`	"Piracy was a big problem in the past, but now it's a relatively minor issue."`
-			`	"Stamped out, may human piracy soon be," says the Arach in a solemn tone of voice. She picks a Saryd for the next question.`
+				`	"There are many independent worlds that are not ruled by one singular government. Some groups there support piracy, but others are peaceful."`
+					goto pirategood
+			`	"Stamped out, may lawlessness soon be," says the Heliarch in a solemn tone of voice. She picks a Saryd for the next question.`
+			label pirategood
+			`	Again, your microphone starts to fizzle out as you attempt to answer. The Heliarch's eight eyes are all on you, and you sense that it might not be a good idea to finish your sentence. The Heliarch selects a Saryd for the next question.`
```

Comment:
I might have misunderstood it, but it seems you can skip the first cutoff? If so, the `Again` would be odd if you did skip it.